patricia_trie/Cargo.toml

```diff
@@ -10,33 +10,17 @@ license = "GPL-3.0"
 elastic-array = "0.10"
 log = "0.3"
 rand = "0.4"
-hashdb = { version = "0.3", path = "../hashdb" }
+hashdb = { version = "0.3.0-beta", path = "../hashdb" }
 parity-bytes = { version = "0.1", path = "../parity-bytes" }
 
 [dev-dependencies]
 env_logger = "0.5"
 ethereum-types = "0.4"
 keccak-hash = { version = "0.1", path = "../keccak-hash" }
-memorydb = { version = "0.3", path = "../memorydb", default-features = false }
-rlp = { version = "0.3.0", path = "../rlp", default-features = false }
+memorydb = { version = "0.3.0-beta", path = "../memorydb", default-features = false }
+rlp = { version = "0.3.0-beta", path = "../rlp", default-features = false }
 trie-standardmap = { version = "0.1", path = "../trie-standardmap", default-features = false }
-triehash = { version = "0.3", path = "../triehash", default-features = false }
+triehash = { version = "0.3.0-beta", path = "../triehash", default-features = false }
 parity-bytes = { version = "0.1.0", path = "../parity-bytes" }
-
```

Comment:
Are these comments no longer relevant?

Reply:
Well, yes and no. The comment is old and the intention was to resolve the problem thanks to some smart reviewers pointing to a solution. That didn’t happen and since I have pretty much learned that no, there is no (much) better way. The crates need proper test helpers. There is an open pr that tries to address that but it’s not truly satisfying either so I’m not pushing it.